content/en/docs/home/contribute/localization.md

```diff
@@ -8,47 +8,87 @@ approvers:
 
 {{% capture overview %}}
 
-We're happy to add localizations (l10n) of Kubernetes documentation to the website!
+The Kubernetes documentation is currently available in [multiple languages](#supported-languages) and we encourage you to add new localizations ([l10n](https://blog.mozilla.org/l10n/2011/12/14/i18n-vs-l10n-whats-the-diff/))!
 
-Localizations must meet the following requirements for _workflow_ (how to localize) and _output_ (what to localize).
+In order for localizations to be accepted, however, they must fulfill some requirements related to [workflow](#workflow) (*how* to localize) and [output](#output) (*what* to localize).
 
 {{% /capture %}}
 
 {{< toc >}}
 
 {{% capture body %}}
 
-## Workflow  
+## Supported languages
+
+The Kubernetes documentation is currently available in the following languages:
+
+{{< languages-list >}}
 
-All l10n work must be stored and tracked within the [Kubernetes organization](https://github.com/kubernetes).
+## Workflow  
 
-### Basis for localizations
+All Kubernetes documentation for all languages is built from the [kubernetes/website](https://github.com/kubernetes/website) repository on GitHub. You don't need to create or maintain a separate repository to add a localization.
 
-Localizations must source from the English files for the [most recent major release](https://kubernetes.io/docs/home/supported-doc-versions/#current-version).
+### Getting started
 
-**Note:** To find the most recent release's documentation source files:
-1. Navigate to https://github.com/kubernetes/website.
-2. Select the `release-1.X` branch for the most recent version.
+In order to add a new localization of the Kubernetes documentation, you'll need to make a few modifications to the site's [configuration](#configuration) and [directory structure](#new-directory), and then you can get to work [translating documents](#translating-documents)!
 
-    For example, the branch for Kubernetes v1.9 docs is `release-1.9`.
+We'll walk you through this whole process by adding support for the German language.
 
 {{< note >}}
-Source files reside in the `/docs/` directory.
+For an example localization-related [pull request](../create-pull-request) adding Korean localization to the Kubernetes docs, see [this pull request](https://github.com/kubernetes/website/pull/8636) to the [Kubernetes website repo](https://github.com/kubernetes/website).
 {{< /note >}}
 
-### Repository
+To get started, clone the website repo and `cd` into it:
 
-A l10n team will have a repository specifically dedicated to its work, for example: [kubernetes/kubernetes-docs-cn](https://github.com/kubernetes/kubernetes-docs-cn).
+```bash
+$ git clone https://github.com/kubernetes/website
+$ cd website
+```
 
-{{< note >}}
-**Note:** To open a l10n repository, [contact the SIG docs lead](https://kubernetes.slack.com/messages/C1J0BPD2M) on Slack for assistance.
-{{< /note >}}
+### Releases
```

Comment:
This is ambiguous. It reads like the section is about how to release the translated artifacts.  Maybe change it to something like "Selecting Kubernetes Releases".